src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/browser-base/browser-base.component.ts

```diff
@@ -54,7 +55,7 @@ export abstract class BaseBrowserComponent<TEntity extends IEntityBase> extends
     public sortingTypes: ISortItem[];
     public views: IBrowserView[];
     public activeView: IBrowserView;
-    public activeThumbnailSize?: ThumbnailSize = ThumbnailSize.Medium;
+    public activeThumbnailSize: ThumbnailSize | undefined = ThumbnailSize.Medium;
```

Comment:
Get rid of the undefined that doesn't make sense

Reply:
What about this?